File: leetcode/26. Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        for(int j = 0; j < nums.size(); ++j){
            if(j != nums.size() - 1 && nums[j] == nums[j + 1]){
                continue;
            }
            nums[i] = nums[j];
            ++i;
        }
        return i;
    }
};

File: leetcode/26. Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) <= 1:
            return len(nums)
        i = 1
        j = 1
        while j < len(nums):
            if nums[j] != nums[i-1]:
                nums[i] = nums[j]
                i += 1
            j += 1
        return i
